src/server/graphql/models/Meeting/meetingMutation.js

```diff
@@ -1,20 +1,20 @@
 import {Meeting} from './meetingSchema';
-import r from '../../../database/rethinkdriver';
-import {isLoggedIn} from '../authorization';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
+// import {isLoggedIn} from '../authorization';
```

Comment:
I see you commented this out because it's unused below. @mattkrick, was the below commenting out intentional?